assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -26,15 +31,20 @@ const loadStripe = () => new Promise((resolve) => {
 
 });
 
-export const setup = (token: Function, closed: Function) => loadStripe().then(() => {
+export const setup = (
+  state: StripeState,
+  token: Function,
+  closed: Function,
+) => loadStripe().then(() => {
 
   stripeHandler = window.StripeCheckout.configure({
     name: 'Guardian',
     description: 'Please enter your card details.',
+    allowRememberMe: false,
     key: 'pk_test_Qm3CGRdrV4WfGYCpm0sftR0f',
```

Comment:
This is the test public key ('publishable key') for the Stripe API, taken from [`membership-common`](https://github.com/guardian/membership-common/blob/master/conf/touchpoint.DEV.conf). In terms of privacy, it should be ok to version control it here, however it may be that there's a better place to keep it in the codebase in the interests of code cleanliness.